src/Tribe/Assets.php

```diff
@@ -154,13 +174,17 @@ public function enqueue( $forcibly_enqueue = null ) {
 			} else {
 				wp_enqueue_style( $asset->slug );
 			}
+
+			$this->assets[ $key ]->already_enqueued = true;
```

Comment:
👍 Done. Didn't realize objects were references in a foreach assignment. Thanks for the tip.